homepage.css

```diff
@@ -0,0 +1,68 @@
+.category-cards {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.category-cards .card {
+  max-width: 20%;
+  object-fit: cover;
+  text-align: center;
+  cursor: pointer;
+  overflow-wrap: break-word;
+}
+
+.category-cards .card:hover {
+  transform: scale(1.05);
+}
+
+.category-cards .card .card-title {
```

Comment:
I've got a question, whether the extra specificity is required here?

Reply:
Actually .card and .card-title are classes from component library(for all cards) which I didn't want to modify directly and hence this selector. I am not sure if this is the the right approach. Let me know your opinion.